### ISSUE TITLE
[ContextualMenu][Accessibility]: disabled item is still focusable

### ISSUE THREAD

**Issue by fsteffi** [OfficeDev/office-ui-fabric-react#11667]
Environment Details:​​
Browser Versions: Microsoft Edge 44.18362.387.0,Chrome Version 78.0.3904.97
Windows Version: Windows10​​​

Repro Steps:​​
1. Hit the application URL
2. Repeat the same steps as ( Select application as "Sales hub" from the list of application--> From the change area which is present in the bottom select "App Settings"-->Navigate to "Site Map" and then Navigate to "Goal Management" and select "Forecast Configuration")
3. Fast Track Your Forecast Screen will appears.
4. Click on "Org chart forecast"
5. Forecast Configuration screen will appears
6. Fill all the mandatory field and click on "next button".
7. Again click on next button.
8.Navigate using tab key to more option button .

Actual:​
On navigating using tab key to more option button hit enter , use tab key and navigate through the items , navigation is visible on disabled items "delete".

Same issue observed in the following page:
1.Configure (focus on disabled button save and close on tab key navigation)
2.Select an option set from the list 
3.additional filters

Expected:​
Tab key navigation should not move to the disabled item.

User Impact:​
Keyboard users will assume it as actionable.

[1639597_A11y_D365ForecastConfiguration_Layoutandcolumns_Keyboard.pptx](https://github.com/OfficeDev/office-ui-fabric-react/files/4044782/1639597_A11y_D365ForecastConfiguration_Layoutandcolumns_Keyboard.pptx)

**Comment by xugao** [comment]
@fsteffi - your repro details are your application-specific. in general, pls make your repro simple and minimal.

**Comment by xugao** [comment]
@fsteffi - this behavior is by design. If the items are not focusable, a blind user potentially will never know that the control is there in the first place. Additionally, if disabled items are not focusable, it breaks the muscle memory that accessible users (and power users for that matter) build for executing controls
For example, users learn to press the downArrow key x times when this menu is open to execute a specific control, if that changes because an item above it is disabled then they will potentially accidentally execute an incorrect control.

Let me know if you object

**Comment by fsteffi** [comment]
Hi @xugao it’s allowable for disabled items to receive keyboard focus per the MAS, even if it’s not a good idea.
So can close the bug
